examples/app.py

```diff
@@ -0,0 +1,30 @@
+from distutils.log import error
+from snowplow_tracker import Tracker, Emitter, Subject
+import sys
+
+
+def get_url_from_args():
+    if len(sys.argv) != 2:
+        raise ValueError("Collector Endpoint is required")
+    return sys.argv[1]
+
+
+def main():
+    collector_url = get_url_from_args()
+
+    e = Emitter(collector_url)
+
+    s = Subject().set_platform("pc")
+    s.set_lang("en").set_user_id("test_user")
+
+    t = Tracker(e, s)
+
+    print("Sending events to " + collector_url)
+
+    t.track_page_view("https://www.snowplowanalytics.com", "Homepage")
+    t.track_page_ping("https://www.snowplowanalytics.com", "Homepage")
+    t.track_link_click("https://www.snowplowanalytics.com")
```

Comment:
Can we also add an example for tracking a self-describing and structured event? I think some people might use this example app for inspiration and it would be good to show them how to track the custom events.